src/hive/utils.js

```diff
@@ -0,0 +1,13 @@
+export function isHiveValueValid(value) {
+  if (value == null) {
```

Comment:
why `null` is invalid value?

Reply:
There server error when I provide null

Reply:
Yeah. The server throws an error with all methods except SortedSet methods set and add. But there we don't use this validation

Reply:
ok